docs/DOC_WRITING_GUIDELINES.md

```diff
@@ -0,0 +1,16 @@
+# Documentation Writing Guidelines
+
+## Best practices
+
++ Check the spelling and grammar, even if you have to copy and paste from external source.
++ Use simple sentences. Easy-to-read sentences mean the reader can quickly use the guidance you share.
++ Try to express your thoughts in a concise and clean way.
++ Don't abuse `code` format when talking in plain English.
++ Follow Google developer documentation style [guide](https://developers.google.com/style).
++ Check the meaning of the words in Microsoft a-z word list term [collections](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/a/above) (use the search input!).
```

Comment:
This link points to a specific entry. There is no link for the index of this section so we should probably start with the first page of this section.
```suggestion
+ Check the meaning of words in Microsoft's [A-Z word list and term collections](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/accessibility-terms) (use the search input!).
```